jetty9-base/pom.xml

```diff
@@ -28,6 +28,18 @@
   <artifactId>jetty9-base</artifactId>
   <packaging>jar</packaging>
 
+  <dependencyManagement>
```

Comment:
Why not import it in the top-level `pom.xml`?

Reply:
Will do, new to maven so I didn't know one could do that.